codegen-core/common-test-models/constraints.smithy

```diff
@@ -497,6 +543,18 @@ integer MaxRangeInteger
 @range(min: 69, max: 69)
 integer FixedValueInteger
 
+@range(min: -0, max: 10)
+integer RangeShort
```

Comment:
Am I misunderstanding, isn't this (and below) supposed to be `short RangeShort`?

https://smithy.io/2.0/spec/simple-types.html#short

Reply:
Ups, yes - fixed in https://github.com/awslabs/smithy-rs/pull/2034/commits/3fd6655869effd811e2c242c40a07d9db11a9932